app/scripts/controllers/transactions/index.js

```diff
@@ -300,6 +300,8 @@ export default class TransactionController extends EventEmitter {
       txParams: normalizedTxParams,
     });
 
+    txMeta.dappSuggestedGasFees = null;
```

Comment:
Do we want to add `dappSuggestedGasFees` to what `generateTxMeta` returns?

Reply:
Smart move. DId that and added some tests.